### ISSUE TITLE
EasyEffects documentation missing

### ISSUE THREAD

**Issue by winny-** [NixOS/nixpkgs#266782]
### Describe the bug
EasyEffects documentation appears to be inaccessible or missing.

### Steps To Reproduce
1. `nix run nixpkgs#easyeffects`
2. Type `F1` or click the hamburger menu -> Manual.
3. Observe no documentation found.

### Expected behavior
Documentation loads in documentation browser.

### Screenshots

![image](https://github.com/NixOS/nixpkgs/assets/3893828/c26686f9-522b-43c4-a054-aebdd30189f3)

### Notify maintainers

Oh snap, I think it's orphaned?

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
winston@icarus ~ $ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.1.55, NixOS, 23.05 (Stoat), 23.05.20230927.e3f57c8`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.13.5`
 - nixpkgs: `/nix/store/x87x8b1dfvmzwgapjqphkgw27am77b37-source`
```

**Comment by bjornfor** [comment]
Seems to be fixed in NixOS 23.11 (tested with GNOME desktop):

![image](https://github.com/NixOS/nixpkgs/assets/133602/d82e60cd-dc9e-42a9-a609-048d3cdd84f7)

If you still observe missing docs after distro upgrade, ping me and I'll re-open.

**Comment by bjornfor** [comment]
> nix run nixpkgs#easyeffects

Perhaps this is a generic issue with apps not being "installed"? (I have easyeffects in `environment.systemPackages`.)
